test/integration/src/py/test_ebrelayer_replay.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import os
+import time
+
+import burn_lock_functions
+import test_utilities
+from burn_lock_functions import EthereumToSifchainTransferRequest
+from integration_env_credentials import sifchain_cli_credentials_for_test
+from test_utilities import get_required_env_var, get_shell_output, SifchaincliCredentials
+
+
+def build_request(
+        smart_contracts_dir,
+        ethereum_address,
+        solidity_json_path,
+) -> (EthereumToSifchainTransferRequest, SifchaincliCredentials):
+    new_account_key = get_shell_output("uuidgen")
+    credentials = sifchain_cli_credentials_for_test(new_account_key)
+    new_addr = burn_lock_functions.create_new_sifaddr(credentials=credentials, keyname=new_account_key)
+    credentials.from_key = new_addr["name"]
+    request = EthereumToSifchainTransferRequest(
+        sifchain_address=new_addr["address"],
+        smart_contracts_dir=smart_contracts_dir,
+        ethereum_address=ethereum_address,
+        ethereum_private_key_env_var="ETHEREUM_PRIVATE_KEY",
+        bridgebank_address=get_required_env_var("BRIDGE_BANK_ADDRESS"),
+        ethereum_network=(os.environ.get("ETHEREUM_NETWORK") or ""),
+        amount=9 * 10 ** 18,
+        solidity_json_path=solidity_json_path
+    )
+    return request, credentials
+
+
+def test_transfer_eth_to_ceth_using_replay_blocks(
+        integration_dir,
+        smart_contracts_dir,
+        solidity_json_path,
+        source_ethereum_address,
+        validator_address,
+        ensure_relayer_restart
+):
+    starting_block = test_utilities.current_ethereum_block_number(smart_contracts_dir)
+    logging.info("stopping ebrelayer")
+    test_utilities.get_shell_output("pkill -9 ebrelayer || true")
+    request, credentials = build_request(smart_contracts_dir, source_ethereum_address, solidity_json_path)
+    logging.info("(no transactions should happen without a relayer)")
+    # test_utilities.whitelist_token(request.ethereum_symbol, request.smart_contracts_dir, True)
+    logging.info(f"send {request.amount / 10 ** 18} eth ({request.amount} wei) to {request.sifchain_address}")
+    test_utilities.send_from_ethereum_to_sifchain(request)
```

Comment:
Let's assert right here after 50 blocks that the tx to sifchain did not go through and then move on to the next steps of this test.

Reply:
Done